src/main/webapp/app/exam/manage/exams/exam-mode-picker/exam-mode-picker.component.spec.ts

```diff
@@ -22,18 +22,14 @@ describe('ExamModePickerComponent', () => {
             .then(() => {
```

Comment:
Please fix the code style in this file and remove the unused import `import { input } from '@angular/core';`